### DISCUSSION TITLE
Non-server support

### DISCUSSION THREAD

**Discussion by i1xiu** [StoneLabs/working-graves#20] [General]
Are you working on making this able to work for non dedicated servers, cuz I really like how this mod has worked so far, its awesome, but I like playing in singleplayer and its kinda tedious to have to start up my dedicated server just so I can be graced by this mods eloquent beauty every time I die. (I really like graveyards, especially when they can actually work)

**Comment by StoneLabs** [comment]
> so I can be graced by this mods eloquent beauty
haha thanks 

I can take a look when i find some time. It shouldn't be too difficult.

**Reply by i1xiu** [reply]
WOW THANK YOU SO MUCH I DIDNT EXPECT A RESPONSE SO SOON YOUR AMAZING

**Reply by StoneLabs** [reply]
See #21 ; Should be working already.
